packages/angular/cli/models/json-schema.ts

```diff
@@ -0,0 +1,106 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+import { JsonObject, JsonValue, parseJson } from '@angular-devkit/core';
+import * as jsonSchemaTravese from 'json-schema-traverse';
+import { Option, OptionSmartDefault } from './command';
+
+export async function convertSchemaToOptions(schema: string): Promise<Option[]> {
+  const options = await getOptions(schema);
+
+  return options;
+}
+
+function getOptions(schemaText: string, onlyRootProperties = true): Promise<Option[]> {
```

Comment:
Same with this.

Reply:
This is only an local function used by `convertSchemaToOptions`